modules/plugin/shapefile/src/main/java/org/geotools/data/shapefile/ShapefileDataStore.java

```diff
@@ -219,6 +221,14 @@ void setMaxDbfSize(long maxDbfSize) {
         this.maxDbfSize = maxDbfSize;
     }
 
+    public boolean isTryCpgFile() {
```

Comment:
ShapefileDataStore is really old and many public methods lack Javadoc. That said, recently we are trying to give new public API minimal docs. Could you add javadocs to these new getter and setter?